public/toolkit/styles/molecules/_dropdowns.scss

```diff
@@ -151,5 +151,9 @@ button.dropdown-select {
 }
 
 .unit-dropdown__control {
-  border: none;
+  border: none !important;
+}
+
+.unit-dropdown__menu-list {
+  overflow-x: hidden !important;
```

Comment:
I'm wondering if this is the best solution for this, is it possible to set the width of the menu to not have overflow? This seems like it could cut content out in some cases?

Reply:
I had the same concern, but dismissed it. Since you also raised the concern maybe it's worth figuring out the root of the problem, which is probably a width thing

Reply:
ok, I can give it another look